builtin/providers/aws/resource_aws_cloudwatch_event_target.go

```diff
@@ -29,7 +30,7 @@ func resourceAwsCloudWatchEventTarget() *schema.Resource {
 
 			"target_id": &schema.Schema{
 				Type:         schema.TypeString,
-				Required:     true,
+				Optional:     true,
 				ForceNew:     true,
```

Comment:
Hi @iceycake do you still want to force a new resource if the target_id changes?